mono/metadata/appdomain.c

```diff
@@ -130,9 +130,13 @@ mono_domain_asmctx_from_path (const char *fname, MonoAssembly *requesting_assemb
 static void
 add_assemblies_to_domain (MonoDomain *domain, MonoAssembly *ass, GHashTable *hash);
 
+#if ENABLE_NETCORE
+
```

Comment:
Unrelated.

Reply:
Moved to https://github.com/mono/mono/pull/16622

Will undo here.